getting-started/templates/systemlink-values.yaml

```diff
@@ -1082,6 +1082,39 @@ nbexecservice:
         # trustedCertificateSecret:
         #   secretName: S3Certificate
         #   key: cert
+    workflow:
+      run:
+        # Default resource allocation for execution pods
+        resources:
+          requests:
+            cpu: "0.1"
+            memory: 2Gi
+          limits:
+            memory: 2Gi
+        # Resource profiles enable different resource allocations for each execution. Profile names are fixed and cannot be modified.
```

Comment:
I'd like to add a point about how the profile is selected in the API call when creating an execution. Otherwise this could leave the user with question about where they need to specify in Helm (or else where) what profile to be used.